### DISCUSSION TITLE
How to disable transparency of the sidebar?

### DISCUSSION THREAD

**Discussion by MrGreen-22** [AutoDarkMode/Windows-Auto-Night-Mode#904] [Q&A] [answered]
Hi. I don't like the transparency effect and have disabled it in Windows settings, but the app itself has a transparent sidebar and I didn't find an option in the settings to disable it. Is there a way to do this?

**Comment by Jay-o-Way** [comment, marked as answer]
At the moment, effort is being made to port to WinUI 3. This should be handled by WinUI.
